lib/state/ui/reducer.ts

```diff
@@ -86,7 +86,15 @@ const showNoteInfo: A.Reducer<boolean> = (state = false, action) => {
 
     case 'App.toggleNavigation':
       return false;
+    default:
+      return state;
+  }
```

Comment:
maybe belongs in another PR? It looks like we introduced this bug and didn't notice it because `undefined` collapses into `false`